assets/sass/patterns/organisms/content-header.scss

```diff
@@ -292,25 +292,24 @@
 
 }
 
+.content-header__component-assets {
+  display: grid;
+  grid-template-rows: auto auto auto;
```

Comment:
I believe that this could be shortened to: `grid-template-rows: auto;`

Reply:
I am changing this.